identity-service/src/routes/solana.js

```diff
@@ -21,11 +24,29 @@ const isValidInstruction = (instr) => {
   return true
 }
 
-solanaRouter.post('/relay', handleResponse(async (req, res, next) => {
+solanaRouter.post('/relay', authMiddleware, handleResponse(async (req, res, next) => {
   const redis = req.app.get('redis')
   const libs = req.app.get('audiusLibs')
+  let optimizelyClient
+  let socialProofRequiredToSend = true
 
   let { instructions = [], skipPreflight, feePayerOverride } = req.body
+  try {
+    optimizelyClient = req.app.get('optimizelyClient')
+    socialProofRequiredToSend = getFeatureFlag(optimizelyClient, FEATURE_FLAGS.SOCIAL_PROOF_TO_SEND_AUDIO_ENABLED)
+  } catch (error) {
```

Comment:
I think this won't throw (just return default instead of erroring), but no harm here. log is great